.appveyor.yml

```diff
@@ -0,0 +1,18 @@
+environment:
+  fast_finish: true
```

Comment:
I think this should be under `matrix` (not this one under environment, just add it to the bottom), e.g:

```yml
matrix:
  fast_finish: true
```